cordova-lib/spec-plugman/platforms/windows.spec.js

```diff
@@ -192,34 +203,39 @@ beforeEach(function () {
 
                 projectsAddedToSpies.forEach(function (spy) {
                     expect(spy).toHaveBeenCalled();
+                    spy.calls.reset();
```

Comment:
actually, not sure. Maybe try without and see what happens? You have two more resets in this file below